src/gateway.h

```diff
@@ -35,10 +35,20 @@ struct gateway {
 	uint64_t protocol;           /* Protocol format version */
 	uint64_t client_id;
 	struct id_state random_state; /* For generating IDs */
+	/* handle_exec_sql uses this to defer some work to the next loop
```

Comment:
Same comment about `handle_exec_sql` being internal and this being part of the external documentation.